src/promptflow/promptflow/batch/_csharp_executor_proxy.py

```diff
@@ -23,115 +28,38 @@ def create(
         *,
         connections: Optional[dict] = None,
         storage: Optional[AbstractRunStorage] = None,
-    ) -> "AbstractExecutorProxy":
+        **kwargs,
+    ) -> "CSharpExecutorProxy":
         """Create a new executor"""
-        working_dir = working_dir or flow_file.parent
-
-        from pythonnet import load
-
-        # pythonnet.load must be called before import clr
-        load("coreclr")
-        import clr
-
-        with _change_working_dir(working_dir):
-            sys.path.append(working_dir.as_posix())
-            clr.AddReference("Promptflow")
-            sys.path.pop()
-            from Promptflow.Executor import YamlExecutor
-
-            connection_provider_url = ""
-            executor = YamlExecutor(
-                flow_file.read_text(encoding="utf-8"), connection_provider_url, working_dir.as_posix()
-            )
-            return cls(executor)
+        port = cls.find_available_port()
+        command = ["dotnet", EXECUTOR_SERVICE_DLL, "-p", port, "-y", flow_file, "-a", ".", "-c", "", "-l", ""]
```

Comment:
give full name for `-c` and `-l` given they're magic value for now?

Reply:
Updated.